.github/CODEOWNERS

```diff
@@ -24,7 +24,17 @@
 /src/azure-cli/azure/cli/command_modules/advisor/ @Prasanna-Padmanabhan
 /src/azure-cli/azure/cli/command_modules/apim/ @kevinhillinger @jonlester
 /src/azure-cli/azure/cli/command_modules/appconfig/ @shenmuxiaosen @avanigupta @zhoxing-ms @jsntcy @yanzhudd
-/src/azure-cli/azure/cli/command_modules/appservice/ @panchagnula @jsntcy @zhoxing-ms @yanzhudd
+/src/azure-cli/azure/cli/command_modules/appservice/ @jsntcy @zhoxing-ms @yanzhudd
+# owners for 'az webapp'
+/src/azure-cli/azure/cli/command_modules/appservice/ @seligj95
+# owners for 'az functionapp'
+/src/azure-cli/azure/cli/command_modules/appservice/ @pamamounelsayed @kaibocai @shreyas-gopalakrishna @kamperiadis @apawast
+# owners for 'az logicapp'
+/src/azure-cli/azure/cli/command_modules/appservice/ @archidda
+# owners for 'az staticwebapp'
+/src/azure-cli/azure/cli/command_modules/appservice/ @sulabh-msft @vivekjilla @joslinmicrosoft @mkarmark 
+# owners for 'az appservice ase'
+/src/azure-cli/azure/cli/command_modules/appservice/ @madsd @hamster-huey
```

Comment:
I'm not sure if the same module can be split into multiple records. I suggest that the owner information corresponding to these sub-modules can be written in the comments instead of splitting into multiple records

Reply:
Hi @haroonf,
I think GitHub code owners doesn't support this feature.
We can close this PR, but our bot can support this.
[This](https://github.com/Azure/azure-cli/pull/24973) is the command group based routing rules we implemented for appservice.
I think we just need to make a few more adjustments based on this PR?